Admin/AbstractAdmin.php

```diff
@@ -1598,7 +1598,7 @@ public function getSubject()
     {
         if ($this->subject === null && $this->request) {
             $id = $this->request->get($this->getIdParameter());
-            if (!preg_match('#^[0-9A-Fa-f\-]+$#', $id)) {
+            if (($id === null) || !preg_match('#^[0-9A-Fa-f\-]+$#', $id)) {
```

Comment:
Ohh ... I got confused.

Execution didn't actually enter this branch b/c `$this->subject` is an instance of my Entity.

It's my bad. Please ignore this PR. I'll close it.